### ISSUE TITLE
JetReview x Jet Smart Filters

### ISSUE THREAD

**Issue by noxdiurna** [CrocoBlock/suggestions#1829]
Hi.

I just realized that there is no specific variable query for JetReview average rating that can be integrated to Jet Smart Filters. I think this is awkward situation as both of these plugins have been recently updated pretty much at the same time. 

I requested the tech team from Crocoblock as I couldn't believe that this was not integrated but apparently, that is the case?

I first had a person from Crocoblock on chat who suggested that it may be the review "slug" but that doesn't seem to work. Apparently the tech team confirms that the functionality jus just not there.

Unless I'm mislead, please integrate JetReview Review listing average rating to Jet Smart Filters.

**Comment by arekuki** [comment]
+1

**Comment by noxdiurna** [comment]
Anyone? Doesn't this shock anyone?

**Comment by ghost** [comment]
Did you find any solution?

**Comment by noxdiurna** [comment]
Nope. I erased JetReviews and decided use another plugin. I would have loved to have ajax review with robust feedback features such as JetReviews on my website but not being able to relate to product rating and only allow certified buyers to leave comments got me change my mind for now.

**Comment by ghost** [comment]
Can you tell me which another plugin you used? can it create filters as well?

**Comment by sara-kariv** [comment]
> Nope. I erased JetReviews and decided use another plugin. I would have loved to have ajax review with robust feedback features such as JetReviews on my website but not being able to relate to product rating and only allow certified buyers to leave comments got me change my mind for now.

which another plugin you used?

**Comment by aalexandr21** [comment]
https://github.com/Crocoblock/suggestions/issues/2099#issuecomment-1681875771
